src/pages/home/library/SongList.tsx

```diff
@@ -288,7 +301,13 @@ export default function SongList({ totalCountOfSongs, query }: SongListProps) {
             return (
               <TableRow
                 onClick={
-                  isSongStale ? undefined : () => handleSongPlayPause(song)
+                  isSongStale
+                    ? undefined
+                    : (event) =>
+                        handleRowClick(event, {
+                          songId: song.id,
+                          hasStartedMintingProcess,
+                        })
```

Comment:
Not a required change but to improve the readability of this would it help to refactor the isSongStale into the handleRowClick function or have a helper function so event is implicitly passed?